src/resources/metrics.py

```diff
@@ -0,0 +1,36 @@
+from flask_restful import Resource
+from src.model.metric import Metrics
+from src.model.pre_config import PreConfig
+from flask import Flask, request, jsonify
+
+
+class PreConfigMetrics(Resource):
+    def post(self):
+
+        data = request.get_json(force=True)
+
+        id_wanted = data["id_wanted"]
+
+        if PreConfig.objects(_id=id_wanted) == {}:
+            print("Id not found!")
+        else:
+            print("Id founded!")
```

Comment:
Se uma request for mandada passando um id de pré configuração inválido o que vai acontecer?

Lembrem-se que a requisição vem de algum lugar pela rede e o protocolo que é usado para essa conversa é o HTPP. Usar print dizendo que encontrou e não encontrou não chega na pessoa que fez a request (não vai na resposta da requisição). As requisições e respostas já são gravadas nos logs da aplicação tbm, então esse print aqui é desnecessário.